### DISCUSSION TITLE
Modifying data series color

### DISCUSSION THREAD

**Discussion by dgwyer** [observablehq/plot#983] [Q&A] [answered]
I have a basic scatter plot (link below). Both the fill and stroke colors are generated from the `group` column. I'd like to be able to modify the stroke color so that it's a little darker than the fill. I've looked in the docs but haven't found a way to access the color. Hopefully this is possible.

https://observablehq.com/@dgwyer/scatter-plot

**Comment by Fil** [comment, marked as answer]
Try moving the _strokeOpacity: 0.5, fillOpacity: 0.2_ part to the dot mark's options

**Comment by dgwyer** [comment]
Thanks, that worked: https://observablehq.com/@dgwyer/scatter-plot

I wonder if, in the future, we might be able to access the actual color before it's rendered so we could also darken/lighten it rather than just change the opacity. I think opacity changes are enough in most cases but if you have a frame bg color set then you'd want to tweak the series color rather than make it semi-transparent.

**Reply by Fil** [reply]
Here's an example https://observablehq.com/@observablehq/plot-colorcontrast-custom-transform
